server/src/main/kotlin/suwayomi/tachidesk/manga/impl/update/Updater.kt

```diff
@@ -53,7 +53,7 @@ class Updater : IUpdater {
     private var currentUpdateTaskId = ""
 
     init {
-        serverConfig.subscribeTo(serverConfig.globalUpdateInterval, ::scheduleUpdateTask)
+        serverConfig.subscribeTo(serverConfig.globalUpdateInterval, ::scheduleUpdateTask, ignoreInitialValue = false)
```

Comment:
The interval also gets started in the ServerConfig
I think I did it that way for the same reason as for the backup interval